config/dev.exs

```diff
@@ -39,7 +39,7 @@ config :apr, :basic_auth, [
 config :phoenix, :stacktrace_depth, 20
 
 config :kafka_ex,
-  brokers: [{"192.168.99.100", 9092}],
-  consumer_group: System.get_env("KAFKA_CONSUMER_GROUP") || "kafka_ex",
-  disable_default_worker: true,
+  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
+  consumer_group: System.get_env("KAFKA_CONSUMER_GROUP") || "kafka_ex_local",
```

Comment:
We might want to also append the USER env var as a default here so we each get our own consumer group in development, i.e. `"kafka_ex_local_#{System.get_env("USER")}"`